### PR TITLE
Add Exception Manager to Concourse

### PR DESCRIPTION
**Pull request by NickGrantX**
# Motivation and Context
We need an exception manager to help us deal with unprocessable messages with zero downtime.

# What has changed
Added Exception Manager to the pipeline.

# How to test?
Fly the pipeline and let her rip!

# Links
Trello: https://trello.com/c/f7LmzvJm

### CODE REVIEW

**Comment by NickGrantX** [comment]
[SPIKE: zero downtime message quarantine and error log reduction (5 days)](https://trello.com/c/f7LmzvJm/1244-spike-zero-downtime-message-quarantine-and-error-log-reduction-5-days)

**Comment by NickGrantX** [comment]
[Exception Manager v1.0](https://trello.com/c/2rkYNAMa/1266-exception-manager-v10)

File: pipelines/ci-sit-kubernetes-pipeline.yml [hunk 1]
```diff
@@ -1032,3 +1076,35 @@ jobs:
...
+      SERVICE_ACCOUNT_JSON: ((gcp.service_account_json))
+      GCP_PROJECT_NAME: ((sit-gcp-project-name))
+      KUBERNETES_CLUSTER: ((sit-kubernetes-cluster-name))
+      KUBERNETES_DEPLOYMENT_NAME: exception-manager
+      KUBERNETES_SELECTOR: app=exception-manager
+      KUBERNETES_FILE_PATH: kubernetes-repo/microservices
+      KUBERNETES_FILE_PREFIX: exception-manager
+      WAIT_UNTIL_AVAILABLE_TIMEOUT: 200s
+    input_mapping: {
+      docker-image-resource: cexception-manager-docker-latest,
```
Comment (ryangrundy7) [inline pipelines/ci-sit-kubernetes-pipeline.yml]: Spelling mistake: `cexception `

**Review by ryangrundy7 — CHANGES_REQUESTED** [review]
Looks good, just a mistake for one of the docker resources

**Review by ryangrundy7 — APPROVED** [review]

**Review by liamtoozer — APPROVED** [review]

**Pull request merged**